tests/hooks/test_dash_guard.py

```
@@ -0,0 +1,295 @@
+exit $EXIT_STATUS
+"""
+
+
+def _run_hook_fragment(
+    staged_md_files: str,
+    is_merge: str = "0",
+    cwd: Path | None = None,
+) -> subprocess.CompletedProcess[str]:
+    """Run the dash-check fragment with controlled env."""
+    return subprocess.run(  # noqa: S603 - controlled command, no user input
+        ["bash", "-c", _HOOK_FRAGMENT],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)